### DISCUSSION TITLE
[Rog Flow x16 2023] How to Mux switch without reboot

### DISCUSSION THREAD

**Discussion by jochenny** [seerge/g-helper#2320] [Q&A] [answered]
I try to switch my GPU mode on my x16 2023 to Ultimate mode, which uses dgpu only,  but it metion out that it will reboot for the operation. I remember that x16 2023 model has a MUX switch, which can switch to GPU Ultimate mode without reboot. Is there any posible to realize the MUX?

**Comment by seerge** [comment, marked as answer]
@jochenny MUX switch always requires reboot. This is how those devices are implemented. 

If you want to assign dGPU to display w/o rebooting, you can just select ``Standard`` and then set NVidia Control panel - to Nidia GPU only (instead of Optimus)

**Reply by jochenny** [reply]
Thank a lot! I misunderstood what MUX mean. I'm not a native speaker, so when I translated Nvdia advanced optimus in my language by google, it became MUX. Now I know what Armoury Crate do in this part. Thank you again!

**Comment by IceStormNG** [comment]
What you mean is probably advanced Optimus. Open Nvidia Control panel and select "Nvidia GPU only". This will toggle the multiplexer and doesn't need a reboot.

**Reply by jochenny** [reply]
Yes! You are right, it's my fault that I misunderstood the MUX and advanced Optimus, thank you very much!
